mathesar/database/types/install.py

```diff
@@ -0,0 +1,8 @@
+from sqlalchemy import text
+from mathesar.database.types import constants as c
+
+def create_type_schema(engine):
+    query = f"""CREATE SCHEMA IF NOT EXISTS "{c.TYPE_SCHEMA}";"""
```

Comment:
We already have `create_schema` function in `mathesar/database/schemas.py`

Reply:
oops, I'll change that